### ISSUE TITLE
Lineage class does not allow for np.mean

### ISSUE THREAD

**Issue by Marius1311** [theislab/cellrank#248]
I'm slicing a lineage class object to get values restricted to one cluster and I would like to compute the mean along axis 0, i.e. the mean probability per cluster:
`np.mean(adata.obsm['to_final_states'][mask, :], axis=0)`

```pytb
---------------------------------------------------------------------------
IndexError                                Traceback (most recent call last)
~/miniconda3/envs/cellrank/lib/python3.6/site-packages/IPython/core/formatters.py in __call__(self, obj)
    343             method = get_real_method(obj, self.print_method)
    344             if method is not None:
--> 345                 return method()
    346             return None
    347         else:

~/Projects/cellrank/cellrank/tools/_lineage.py in _repr_html_(self)
    419             )
    420         else:
--> 421             body = "".join(format_row(i) for i in range(self.shape[0]))
    422 
    423         cells = "cells" if self.shape[0] > 1 else "cell"

~/Projects/cellrank/cellrank/tools/_lineage.py in <genexpr>(.0)
    419             )
    420         else:
--> 421             body = "".join(format_row(i) for i in range(self.shape[0]))
    422 
    423         cells = "cells" if self.shape[0] > 1 else "cell"

~/Projects/cellrank/cellrank/tools/_lineage.py in format_row(r)
    389                 f"{super(type(self), self).__getitem__((r, c)):.06f}"
    390                 f"</td>"
--> 391                 for c in range(self.shape[1])
    392             )
    393             return f"<tr>{cells}</tr>"

IndexError: tuple index out of range
Lineage([0.09438334, 0.39487063, 0.51074603],
  names([goblet cells, ciliated cells, AT2 cells]))
```
So I guess it returns the right thing, but still complains?

#### Versions:
cellrank==1.0.0-b.3 scanpy==1.5.1 anndata==0.7.3 numpy==1.18.5 scipy==1.4.1 pandas==1.0.5 scikit-learn==0.22 statsmodels==0.10.2 python-igraph==0.7.1 scvelo==0.2.2.dev51+ga7de78a
> ...

**Comment by michalk8** [comment]
I will have to modify __array_ufunc__ to make this work as intented. This will make it also nicer w.r.t. to the names and colors preserved.

**Comment by michalk8** [comment]
Closed via #250
